### DISCUSSION TITLE
Error on chatcompletion

### DISCUSSION THREAD

**Discussion by mesa57** [open-webui/open-webui#8073] [Issues]
After update to v0.5.0 we get following issue when sending a chat prompt to gpt-4o-mini:

```
INFO:     81.172.216.60:0 - "POST /api/chat/completions HTTP/1.1" 200 OK

ERROR [asyncio] Unclosed client session

client_session: <aiohttp.client.ClientSession object at 0x7ff15042bfd0>

ERROR [asyncio] Unclosed connector

connections: ['deque([(<aiohttp.client_proto.ResponseHandler object at 0x7ff1505dc440>, 564.36978947)])']

connector: <aiohttp.connector.TCPConnector object at 0x7ff15066b910>
```

Reverting to v0.4.8 solved the issue.

**Comment by tjbck** [comment]
Unable to reproduce here, the logs displayed should be more like a warning than an actual error, and should still work as intended. Keep us updated!

**Reply by mesa57** [reply]
The webui gives only a red box with network error.
I have restarted the container: same issue
Restarted the whole server: same issue
Reverted to v0.4.8: problem solved.

So seems clear to me an issue of v.0.5.0

Environment:

VPS (contabo) with  Ubuntu 22.04.5 LTS (GNU/Linux 5.15.0-130-generic x86_64)
Docker version 27.4.1, build b9d17ea
docker network bridged

**Comment by mesa57** [comment]
Ok, found the issue
The new async requests are incompatible with our https (to docker http) proxy.
How can we solve this ?

**Comment by luzhao2** [comment]
I have the same problem.

**Comment by tjbck** [comment]
https://github.com/open-webui/open-webui/issues/8074
